ci/selenium2-tests/009-metadata-fields-search.json

```diff
@@ -0,0 +1,488 @@
+{
```

Comment:
Looks OK, but will need some test steps to check if the desired text/entries are on screen after the search and we haven't got a blank list. That can wait.